cpp/src/parquet/bloom_filter_parquet_test.cc

```diff
@@ -69,4 +71,36 @@ TEST(BloomFilterReader, FileNotHaveBloomFilter) {
   ASSERT_EQ(nullptr, bloom_filter);
 }
 
+TEST(BloomFilterBuilderTest, Basic) {
```

Comment:
Actually I'm not familiar with test writing, it would be great if someone tell me how should I test these...